addon/services/log.js

```diff
@@ -0,0 +1,348 @@
+/**
```

Comment:
Problems on next merges.
For example now there is a branch where "addon/services/flexberry-logging.js" is fixed. If I merge this pull request won't there be merge problems with branch with fixed "addon/services/flexberry-logging.js"?